lib/confluence/set_ops.es6.js

```diff
@@ -94,19 +95,53 @@ foam.CLASS({
 });
 
 foam.CLASS({
+  package: 'org.chromium.mlang.sink',
+  name: 'SetMinus',
+  extends: 'org.chromium.mlang.sink.AsyncSink',
+
+  documentation: `Put incoming objects to delegate iff the same object (by id)
+      is not found in "secondary" (subtrahend) DAO.`,
+
+  listeners: [
+    function onSecondaryFind(sub, o, found) {
+      if (found === null) this.delegate.put(sub, o);
+    },
+  ],
+});
+
+foam.CLASS({
+  package: 'org.chromium.mlang.sink',
+  name: 'Intersect',
+  extends: 'org.chromium.mlang.sink.AsyncSink',
+
+  documentation: `Put incoming objects to delegate iff the same object (by id)
+      is found in "secondary" DAO.`,
+
+  listeners: [
+    function onSecondaryFind(sub, o, found) {
```

Comment:
As discussed offline, I found the logic flow here difficult to follow. I think it could use some documentation.